index.js

```diff
@@ -133,6 +133,19 @@ function serialize(name, val, options) {
     pairs.push('Path=' + opt.path);
   }
 
+  if (null != opt.sameSite) {
+    if(opt.sameSite !== true && opt.sameSite !== false && opt.sameSite !== 'Strict' && opt.sameSite !== 'Lax'){
+      throw new TypeError('option sameSite is invalid');
```

Comment:
Reading the spec, the value is not case-sensitive, so we should accept `strict`, `lax` or any other case combination. Please update this to match the spec :)